tdrs-backend/tdpservice/users/test/test_api.py

```diff
@@ -395,23 +339,193 @@ def test_set_profile_data_extra_field_include_required(api_client, user):
 def test_set_profile_data_missing_last_name_field(api_client, user):
     """Test profile data cannot be set if last name field is missing."""
     api_client.login(username=user.username, password="test_password")
-    response = api_client.patch(
-        "/v1/users/set_profile/",
-        {
-            "first_name": "Heather",
-        },
-    )
+    response = api_client.patch("/v1/users/set_profile/", {"first_name": "Heather", },)
     assert response.status_code == status.HTTP_400_BAD_REQUEST
 
 
 @pytest.mark.django_db
 def test_set_profile_data_missing_first_name_field(api_client, user):
     """Test profile data cannot be set if first name field is missing."""
     api_client.login(username=user.username, password="test_password")
+    response = api_client.patch("/v1/users/set_profile/", {"last_name": "Heather", },)
+    assert response.status_code == status.HTTP_400_BAD_REQUEST
+
+
+@pytest.mark.django_db
```

Comment:
@carltonsmith We talked last week about splitting this file up into smaller files since it's hard to track changes over large files.

Reply:
Yes, we did, and I definitely think it's a great idea. I just think it might be a bit heavier of a lift to attach to this PR. Would you be amenable to me creating a separate issue to break these up?

Reply:
Hmmmm... 

On the one hand, I'm not sure if I'd want this improvement by itself to hold up the entire PR. 

On the other hand, moving new tests to a separate file instead of adding them to the bottom of the existing file should be a very straightforward change.

We did say we want to emphasize quality when it comes to the "iron triangle." 

And this speaks to an element in the [Heuristics](https://github.com/HHS/TANF-app/blob/cb15595d3ad6b7a984f083c8c81edde036315b59/docs/Technical-Documentation/Heuristics.md) doc:

Indicator | Good sign
-- | --
Code has logical organization. | When files become large they are broken out into a folder, and folder structure could be reasonably understood by a developer less-familiar with the project. Non dev: The code is well organized by file and folder.

Reply:
Yeah, that makes sense. My initial thought was to do something that would take more work, but I believe all of these tests are related to the user app, so it shouldn't be as much effort as I was thinking.

Reply:
This is all set 😄

Reply:
Love it, much easier to read!